governance/votes/Executive vote - November 1, 2023.md

```diff
@@ -0,0 +1,92 @@
+
+If this executive proposal passes, the DAO Resolution with the IPFS hash of [QmbrCPtpKsCaQ2pKc8qLnkL8TywRYcKHYaX6LEzhhKQqAw](https://gateway.pinata.cloud/ipfs/QmbrCPtpKsCaQ2pKc8qLnkL8TywRYcKHYaX6LEzhhKQqAw) will be approved. 
+
+### Immunefi Security Core Unit MKR Vesting Transfer
+
+*Authorization*: [Original ratified MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP41)\
+ *Proposal*: [Forum post](https://forum.makerdao.com/t/mip39c3-sp13-removing-is-001/22392) "MIP39c3-SP13: Removing IS-001"
+
+Per its voluntary offboarding proposal, Immunefi Security CU requests payout of the MKR vested 2023-08-05, adjusted to 0.4 FTE. (See [manual repricing](https://forum.makerdao.com/t/mip40c3-sp41-immunefi-security-core-unit-mkr-budget-is-001/10814/3); FTE reduction in [July 2022](https://github.com/makerdao-is/transparency-reporting/blob/main/Monthly%20Budget%20Statements/2022-07.md#fte); adjusted MKR vesting schedule in [August 2022](https://github.com/makerdao-is/transparency-reporting/blob/main/Monthly%20Budget%20Statements/2022-08.md#3-mkr-vesting-overview); and final FTE reduction to 0.4 in [January 2023](https://github.com/makerdao-is/transparency-reporting/blob/main/Monthly%20Budget%20Statements/2023-01.md#fte).) 
+
+If this executive proposal passes, a one-time MKR payment in the amount of **6.34 MKR** will be transferred to the Core Unit's Auditor Wallet ([0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4](https://etherscan.io/address/0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4)).
+
+### Add GUSD and USDP Jar and SwapInputConduit Contracts to Chainlog
+
+*Proposal*: [Forum Post](https://forum.makerdao.com/t/proposed-housekeeping-item-upcoming-executive-spell-2023-11-01/22477) "Proposed Housekeeping Item (Upcoming Executive Spell - 2023-11-01)"\
+*Authorizing Party*: [Protocol Scope Facilitator](http://forum.makerdao.com/t/proposed-housekeeping-item-upcoming-executive-spell-2023-11-01/22477/2)
+
+In [December 2022](https://forum.makerdao.com/t/gusd-institutional-rewards-technical-solution-risk-assessment-and-deployment-details-for-receiving-payments-of-rewards/19055), a GUSD & MakerDAO partnership for institutional rewards was rolled out. In order for MakerDAO to receive the GUSD rewards, two contracts were deployed: the RwaSwapInputConduit2 and RwaJar. Recently, MakerDAO entered into a similar [partnership with Trident](https://vote.makerdao.com/executive/template-executive-vote-usdp-psm-incentives-reth-initial-offboarding-rwa-vaults-reconfiguration-various-parameter-changes-avc-and-ad-compensation-facilitator-and-ecosystem-actor-compensation-spark-proxy-spell-october-11-2023) to receive USDP rewards. This deal also requires deploying the above-mentioned contracts.
```

Comment:
I believe it would be clearer for readers if we linked to the agreement details rather than the executive bundle in this context:

https://forum.makerdao.com/t/usdp-psm-facilitation-incentives/22331

Reply:
I will leave as is, as the Executive Vote includes the link to the Forum post and agreement details are summarized succinctly there.